install/kubernetes/quick-install.yaml

```diff
@@ -469,6 +469,7 @@ spec:
         - mountPath: /var/run/cilium
           name: cilium-run
       restartPolicy: Always
+      priorityClassName: system-node-critical
```

Comment:
I'm not sure where this came from, but it was added automatically. Calling it out in case someone knows if it's harmless or not.

Reply:
/cc @aanm I think we usually drop this hunk manually